preprocessing.py

```diff
@@ -205,17 +205,20 @@ def get_input_tensors(batch_size, tf_records, num_repeats=1,
 
 
 def get_tpu_input_tensors(batch_size, tf_records, num_repeats=1,
-                          shuffle_records=True, shuffle_examples=True,
-                          shuffle_buffer_size=1024,
                           filter_amount=1, random_rotation=True):
+    # TPUs trains on sequential golden chunks to simplify preprocessing and
```

Comment:
This isn't really necessarily true after we do Cloud BT integration.  i think you axe this assert and comment.

Reply:
BT doesn't use this code path, it calls `get_tpu_bt_input_tensors` which makes a supcall to bigtable_input to do most of it's processing.